vault/testing.go

```diff
@@ -1924,7 +1928,7 @@ func (cluster *TestCluster) StartCore(t testing.T, idx int, opts *TestClusterOpt
 	}
 	tcc.Listeners = []*TestListener{
 		{
-			Listener: tls.NewListener(ln, tcc.TLSConfig),
+			Listener: tls.NewListener(ln, tcc.tlsConfig),
```

Comment:
Any reason why these aren't also clones for consistency, even though they shouldn't mutate the config?

Reply:
It didn't even occur to me to do that.  I mean, this is what the tlsConfig is *for*, it's others who access it for their own purposes that should be cloning.